File: pgm/ABC_Java/ABC_370_A.java
import java.io.*;
// import java.math.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final String[] arr = reader.readLine().split (" ");
        final String Yes = "Yes";
        final String No = "No";
        final String Invalid = "Invalid";
        final int L = Integer.parseInt (arr[0]);
        final int R = Integer.parseInt (arr[1]);
        if (1 == L && 0 == R) {
            System.out.println (Yes);
        } else if (0 == L && 1 == R) {
            System.out.println (No);
        } else {
            System.out.println (Invalid);
        }
        return;
    }
}

File: pgm/ABC_Visual_Basic/ABC_370_A.vb
' Option Strict On
' Imports System.Collections.Generic
' Imports System.Globalization
' Imports System.IO
' Imports System.Linq
' Imports System.Text
' Imports System.Text.RegularExpressions

Class Main
    Public Shared Sub Main()
        Dim arr() As String = Console.ReadLine().Split()
        Const Yes As String = "Yes"
        Const No As String= "No"
        Const Invalid As String = "Invalid"
        Const one As Integer = 1
        Const zero As Integer = 0
        Dim L As Integer = Integer.Parse(arr(0))
        Dim R As Integer = Integer.Parse(arr(1))
        IF L = one AndAlso R = zero Then
            Console.WriteLine(Yes)
        ElseIf L = zero AndAlso R = one Then
            Console.WriteLine(No)
        Else
            Console.WriteLine(Invalid)
        End If
        Return
    End Sub
End Class